app/views/local_authorities/show.html.erb

```diff
@@ -1,39 +1,31 @@
 <% content_for :page_title, @authority.name %>
 
-<% if flash[:success] %>
-<div class="alert alert-success"><%= flash[:success] %></div>
-<% end %>
-<% if flash[:danger] %>
-<div class="alert alert-danger"><%= flash[:danger].kind_of?(Array) ? sanitize(flash[:danger].join("<br>")) : flash[:danger] %></div>
-<% end %>
+<%= render "govuk_publishing_components/components/heading", {
+  text: @authority.name,
+  heading_level: 1,
+  font_size: "l",
+  margin_bottom: 5,
+} %>
 
-<% breadcrumb :local_authority, @authority %>
-
-<%= render partial: "shared/local_authority_details", locals: { authority: @authority } %>
-
-<%= render partial: 'link_filter_nav' %>
-
-<table id="links" class="table table-bordered table-service table-with-counts" data-module="filterable-table">
-  <thead>
-    <tr class="table-header">
-      <th colspan='4'>
-        <div class='filter-control'>
-          <%= render partial: 'govuk_admin_template/table_filter_form', locals: { placeholder: 'Search for a council or link here' } %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render "govuk_publishing_components/components/metadata", LocalAuthorityPresenter.new(@authority).metadata(self) %>
+  </div>
+  <div class="govuk-grid-column-one-third">
+    <aside class="app-side__wrapper">
+      <div class="app-side">
+        <div class="app-side__actions">
+          <%= render "govuk_publishing_components/components/list", {
+            items: [
+              link_to("Edit Homepage URL", edit_url_local_authority_path(@authority)),
+              link_to("Download Links", download_links_form_local_authority_path(@authority)),
+              link_to("Upload Links", upload_links_form_local_authority_path(@authority)),
+            ],
+          } %>
```

Comment:
Probably just need to add the `govuk-link` formatting here. You could also add `margin_bottom: 0` to the list component to have equal vertical spacing.

Reply:
Done